modules/modes/ModeDrawArea.js

```diff
@@ -558,4 +559,30 @@ if (choice && choice.distance < SNAP_DIST) {
     this.drawWay = null; // this will trigger a rollback
     this.context.enter('browse');
   }
+
+  /**
+   * _hover
+   * Hover changed cursor styling based one what geometry is hovered
+   */
+  _hover(eventData) {
+    // Get the current context and graph
+    const context = this.context;
+    const graph = context.graph();
+    // Get the target and associated datum
+    const target = eventData.target;
+    const datum = target && target.data;
+    // Check if the datum is an entity in the graph
+    const entity = datum && graph.hasEntity(datum.id);
+    // Get the geometry of the entity, if it exists
+    const geom = entity && entity.geometry(graph);
+    // Change the cursor of the document body based on the geometry type
+    if (geom && geom === 'vertex') {
+      document.body.style.cursor = 'url(/img/cursor/cursor-draw-connect-vertex.png),auto';
```

Comment:
I built and ran this branch, and I noticed that the draw cursors are indeed showing up but they're not offset correctly. All the crosshair-style cursors (the ones that begin with 'cursor-draw') will need to be centered directly on the mouse coordinate, unlike the pointer-type that need to be drawn underneath/to the right.